src/System.Runtime.CompilerServices.VisualC/tests/Configurations.props

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <BuildConfigurations>
+      netcoreapp;
+      uap-Windows_NT;
```

Comment:
Did you debug what dependency is missing? Also it looks a little odd that it is looking for a version 0.0.0.0 of the test assembly.